test/queueservice_test.js

```diff
@@ -2,7 +2,7 @@ suite('queue/QueueService', function() {
   var Promise       = require('promise');
   var slugid        = require('slugid');
   var assert        = require('assert');
-  var QueueService  = require('../lib/queueservice');
+  var QueueService  = require('../src/queueservice');
```

Comment:
It can, but it doesn't. Using mocha and babel with Node builds the source files.